src/shared/modules/dbMeta/dbMetaDuck.ts

```diff
@@ -468,7 +468,7 @@ const clusterRole = (store: any) =>
         return resolve(null)
       }
       bolt
-        .directTransaction(
+        .routedReadTransaction(
```

Comment:
this one we'll need to do as a direct query. otherwise the role will be scrablmed

Reply:
Anyway when running with Neo4j protocol the current role loses it's meaning, as it could have been re-routed on the server side (or the client side)